spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -842,6 +849,7 @@ def _run(self, run_time, run_until_complete=False):
             self._do_mapping(run_time, n_machine_time_steps, total_run_time)
 
         # Check if anything has per-timestep SDRAM usage
+        provide_injectables(self._mapping_outputs)
```

Comment:
if memory serves, its because the SSA now needs it to meet the criteria properly. am trying to verify now. it may be a change thats really from another cherry pick. i never promsied i got all of these perfect

but its worth noting that this whole injection crud is dodgy, coz it allows something in a completely seperate repo to force this. is probabily safer to wrap everything in ASB with a injectable provider anyhow.

Reply:
ok. having checked. it seems to be a cherry pick from timing thats slipped into the app folder stuff. as its in both, and i have vague memories of it needing it for the SSA. Ive also ran a few tests on this pr without them in and it seems to work. so removing